ampd/src/broadcaster/dec_coin.rs

```diff
@@ -238,7 +238,7 @@ mod tests {
 
     #[test]
     fn invalid_try_from_string() {
-        assert!(DecCoin::try_from("10a0u/-xl").is_err());
+        assert!(DecCoin::try_from("10a0u/-xl!").is_err());
```

Comment:
is this just to add another special character or does it have some other reason?

Reply:
Just add the `!`. They allowed a bunch of special characters to match the SDK changes, and our previous "invalid" denom becomes valid.